M2/Macaulay2/packages/FastLinAlg.m2

```diff
@@ -855,6 +936,70 @@ projDim(Module) := opts -> (N1) -> (
     return i;
 );
 
+isGBDone := (myGB) -> (
+    --a temporary function for finding out if a gb computation is done.
+    myStr := status myGB;
+    if (0 < #select("status: done", myStr)) then return true else return false;
```

Comment:
It's a tautology that
```
if X then true else false
```
is equal to X, so the circumlocution is confusing.

Reply:
Fixed.